### ISSUE TITLE
NVDA doesn't read status of live icons in Windows 8 RP

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#2423]
**Reported by csm120 on 2012-06-09 12:31**
When the consumer preview of Windows 8 was out NVDA read the live icons in the start screen. For example if you arrowed over the weather icon you would hear a brief summery of the current weather conditions. These live icons now only speak the name of the icon, and not any content.

**Comment by ehollig** [comment]
@josephsl, is this the case anymore or has this been fixed as stated in https://github.com/nvaccess/nvda/issues/2423#issuecomment-155298560

**Comment by josephsl** [comment]
Hi, Windows 8 release preview? I’d say it should be closed. Thanks.
